src/pages/home/categories-list/categories-list.style.js

```diff
@@ -6,8 +6,9 @@ export const useStyles = makeStyles((theme) => ({
     flexDirection: 'column',
     justifyContent: 'center',
     alignItems: 'center',
-    height: '100vh',
+    height: '45vh',
```

Comment:
It's not good to use VH
I think you need to try to fix this with flex-box

Reply:
Or with maxHeight

Reply:
fixed

(but can you text me why this in not okay to use VH?)